jenkins/gce-custom-images/prepare_gce_image.sh

```diff
@@ -17,17 +17,17 @@ set -e -x
 #
 
 # The versions below don't matter; they are replaced by the Jenkins job
-terra_jupyter_base="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:0.0.19"
-terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.1.1"
-terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:1.0.13"
-terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:1.0.13"
-terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:1.1.1"
-terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.0.23"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.1.2"
-
-welder_server="us.gcr.io/broad-dsp-gcr-public/welder-server:6cfad83"
+terra_jupyter_base="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:0.0.20"
```

Comment:
thought we were saying we're removing these defaults?

Reply:
We removed the Jenkins parameter defaults from `Jenkinsfile`; I think the versions in the `.sh` scripts get replaced regardless

Reply:
What Rob said…

Reply:
Speaking of `prepare….sh` scripts getting replaced…due to the issues mentioned in a [new ticket](https://broadworkbench.atlassian.net/browse/IA-2770?atlOrigin=eyJpIjoiOWVhNzMwNDIyMTNjNDA2YmIzYmQwZWU5MmJmMzA3NzQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) that seem to predate any COS/GPU related work, `welder` and `aou` image versions will not be updated in `prepare….sh` scripts so we need to manually update the `prepare….sh` files when those two image versions change.

Reply:
Good to know! I knew the aou ones didn't get replaced but not welder.